server/daos/authors.js

```diff
@@ -2,18 +2,8 @@ import db from '@database/models';
 
 export const insertAuthor = args => db.books.create(args);
 
-export const updateAuthor = async args => {
-  const mapAuthorUpdateArgs = {
-    name: args.name,
-    country: args.country,
-    age: args.age
-  };
+export const updateAuthor = async ({ id, name, country, age }) => {
+  await db.authors.update({ name, country, age }, { where: { id } });
 
-  const authorResponse = await db.authors.update(mapAuthorUpdateArgs, { where: { id: args.id } });
-
-  console.log('authors response', authorResponse);
-
-  const author = await db.authors.findOne({ where: { id: args.id } });
-
-  return author;
+  return db.authors.findOne({ where: { id } });
```

Comment:
do we need the updated value? this will put unnecessary load on every update

Reply:
yeah we need the updated value but we can remove the find method on every update call